include/avif/avif.h

```diff
@@ -1287,7 +1287,7 @@ typedef struct avifDecoder
     // Internals used by the decoder
     struct avifDecoderData * data;
 
-    // Version 1.0.0 ends here. Add any new members after this line.
+    // Version 1.0.0 ends here.
```

Comment:
Line 1297 below says:

```
    // Version 1.1.0 ends here. Add any new members after this line.
```

So this line should not also say "Add any new members after this line."